### PR TITLE
Bug/sms from

### PR DESCRIPTION
**Pull request by crbrox**
Fix from field in request to SMPP Adapter (#67)

@fgalan @dmoranj  @iariasleon

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -106,9 +106,10 @@ config.smtp = {
  * SMS endpoint options
  */
 config.sms = {
-    URL : 'http://pigeon.hopto.org:443/sms/v2/smsoutbound',
-    API_KEY : '80ce17b0-2232-4788-b506-0efa38e3a22d',
-    API_SECRET: '22c51b7b-3f66-485c-a01b-6dcfc74106be'
+    URL : '',
+    API_KEY : '',
+    API_SECRET: '',
+    from: 'tel:22012;phone-context=+34'
```
Comment (fgalan) [inline config.js]: This attribute is new... it should be explained in the corresponding .md file.

File: config.js [hunk 2]
```diff
@@ -106,9 +106,10 @@ config.smtp = {
  * SMS endpoint options
```
Comment (fgalan) [inline config.js]: Include corresponding line in CHANGES_NEXT_RELEASE file.

File: config.js (hunk 1 above)
Comment (crbrox) [inline config.js]: Fixed in 1a90cecbf6f

File: config.js (hunk 2 above)
Comment (crbrox) [inline config.js]: Fixed in aca1331095c4071

**Comment by fgalan** [comment]
LGTM

**Pull request merged**